x/bitcoin/keeper/keeper.go

```diff
@@ -202,18 +204,38 @@ func (k Keeper) GetTransactionFeeRate(ctx sdk.Context) sdk.Dec {
 	return result
 }
 
-// SetAddress stores the given address information
-func (k Keeper) SetAddress(ctx sdk.Context, address types.AddressInfo) {
-	k.getStore(ctx).Set(addrPrefix.Append(utils.LowerCaseKey(address.Address)), &address)
+// SetAddressInfo stores the given address information
+func (k Keeper) SetAddressInfo(ctx sdk.Context, address types.AddressInfo) {
+	key := depositAddrPrefix.Append(utils.LowerCaseKey(address.Address))
+	k.getStore(ctx).Set(key, &address)
 }
 
-// GetAddress returns the address information for the given encoded address
-func (k Keeper) GetAddress(ctx sdk.Context, encodedAddress string) (types.AddressInfo, bool) {
+// GetAddressInfo returns the address information for the given encoded address
+func (k Keeper) GetAddressInfo(ctx sdk.Context, encodedAddress string) (types.AddressInfo, bool) {
 	var address types.AddressInfo
-	ok := k.getStore(ctx).Get(addrPrefix.Append(utils.LowerCaseKey(encodedAddress)), &address)
+	ok := k.getStore(ctx).Get(depositAddrPrefix.Append(utils.LowerCaseKey(encodedAddress)), &address)
 	return address, ok
 }
 
+// SetDepositAddress associates the specified recipient address with the given deposit address
+func (k Keeper) SetDepositAddress(ctx sdk.Context, recipient nexus.CrossChainAddress, address string) {
```

Comment:
Same as evm, we should avoid saving addresses as string.

Reply:
renamed to `depositAddrPrefix`

Reply:
done